### PR TITLE
Endpoint to get all orders for specific user

### LINKED ISSUE #99: Endpoint to get all orders for specific user
## User story
**Who:** **As a** developer
**What:** **I want to** get all orders made by specific user
**Why:** **so that I can** see it on Order summary page
## Acceptance criteria

The endpoint should show the order number, number of items, date of the order and the amount as shown below.

order-summary

## Implementation Details

Create the GET endpoint for 'api/orders?user={user_id}'

Read about Swagger here https://github.com/HackYourFuture-CPH/fp-class18/#-working-with-api-documentation---swagger and this article should also help you with that https://blog.logrocket.com/documenting-your-express-api-with-swagger/

### PR DESCRIPTION
**Pull request by JiuTak**
# Description

Please provide a short summary explaining what this PR is about.

Fixes #99 

# How to test?
http://127.0.0.1:3000/api/documentation

Check ORDERS 

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [x] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [x] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/server/api/controllers/orders.controller.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+const moment = require('moment-timezone');
+
+const getOrderByUserId = async (user_id) => {
+  if (!user_id) {
+    throw new HttpError('Id should be a number', 400);
```
Comment (Divyajg) [inline src/server/api/controllers/orders.controller.js]: Response not showing 400!
I think the if condition can be more specific like typeof user_id, if user_id doesn't exist.

File: src/server/api/routes/api-router.js [hunk 2]
```diff
@@ -6,6 +6,8 @@ const router = express.Router();
 const modulesRouter = require('./modules.router');
 const productsRouter = require('./products.router');
 const usersRouter = require('./users.router');
+const orderRouter = require('./orders.router');
```
Comment (Divyajg) [inline src/server/api/routes/api-router.js]: it can be ordersRouter ! What do you think?

**Review by Divyajg — CHANGES_REQUESTED** [review]
Code looks good and working good too.
I requested couple of improvisations so that in the below cases it should respond with other than 200 code.

jiu1
jiu2

File: src/server/api/controllers/orders.controller.js (hunk 1 above)
Comment (JiuTak) [reply src/server/api/controllers/orders.controller.js]: fixed :)

File: src/server/api/routes/api-router.js (hunk 2 above)
Comment (JiuTak) [reply src/server/api/routes/api-router.js]: Typo, thanks :)

**Pull request closed without merging**